vm/src/bytecode.rs

```diff
@@ -401,3 +408,23 @@ impl fmt::Debug for CodeObject {
         )
     }
 }
+
+impl From<ast::Varargs> for Varargs {
+    fn from(varargs: ast::Varargs) -> Varargs {
+        match varargs {
+            ast::Varargs::None => Varargs::None,
+            ast::Varargs::NoCapture => Varargs::NoCapture,
+            ast::Varargs::Capture(param) => Varargs::Capture(param.arg),
+        }
+    }
+}
+
+impl Varargs {
+    pub fn from_ast_vararg_ref(varargs: &ast::Varargs) -> Varargs {
```

Comment:
Why not `impl<'a> From<&'a ast::Varargs> for Varargs`?

Reply:
I tried that, but `.into()` didn't work in `compile.rs`, so I sorta just gave up. It does work manually writing `from(&args.varargs)`, though, so I'll change it to be this.